### PR TITLE
Sr/about us

### PR DESCRIPTION
**Pull request by sofiarm21**

### CODE REVIEW

File: src/sections/PastContributors.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+import PastContributorTile from '../components/PastContributorTile'
+import FilledButton from '../components/FilledButton'
+import Headline from '../components/Headline'
+import theme from '../styles/theme';
+
+const Section = styled.div`
+
+    padding-bottom:80px;
+    background: ${props => props.grey ? '#EEEEEE' : 'white'}
```
Comment (otech47) [inline src/sections/PastContributors.js]: Import these from `theme.colors` instead of hardcoding?

File: src/sections/Blog.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ import theme from '../styles/theme';
 
 const Section = styled.div`
     padding-top: 80px;
-    padding-bottom:80px;
+    padding-bottom:30px;
     background: ${props => props.grey ? '#EEEEEE' : 'white'}
```
Comment (otech47) [inline src/sections/Blog.js]: Import these from theme.colors instead of hardcoding?

File: src/sections/PastContributors.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    render() {
+        return (
+            <Section>
+                <Row>
+
+                    <Col xs={10} className='mx-auto'>
+                        <Row>
+                            {this.renderHeadline()}
+                            {this.renderTestimonials()}
+                            {/* {this.renderButton()} */}
```
Comment (otech47) [inline src/sections/PastContributors.js]: remove line

**Comment by sofiarm21** [comment]
> @sofiarm21 can we remove the delay for the tooltips that appear when you hover over the PastContributorTile? It seems okay when you hover on (maybe could be a little faster), but when you hover off the popover/tooltip seems to stay open too long it feels a bit strange.

sure, I'll remove it

**Comment by sofiarm21** [comment]
Ready for review

**Pull request merged**